eqcorrscan/core/match_filter/helpers.py

```diff
@@ -62,7 +61,10 @@ def _spike_test(stream, percent=0.99, multiplier=1e7):
                'FFT prior to cross-correlation.\n' +
                'If you think this spike is real please report ' +
                'this as a bug.')
-        raise MatchFilterError(msg)
+        print(msg)
+        for ID in list_ids:
```

Comment:
variables should not use capitals.
```suggestion
        for _id in list_ids:
```